setup.py

```diff
@@ -32,16 +32,21 @@
 import os
 import ctypes
 import errno
-import time
+from datetime import date
 import sys
-from shutil import copyfile
+from shutil import copyfile, rmtree
 import subprocess
 from setuptools import setup, find_packages, Command
 import setuptools.command.build_py
 
 pkgs = find_packages()
 source_server_file = os.path.abspath('frontend/server/build/libs/server-1.0.jar')
 dest_file_name = os.path.abspath('mms/frontend/model-server.jar')
+extras = {
+    'mxnet_cpu': ['mxnet-mkl==1.2.0'],
+    'mxnet_gpu': ['mxnet-cu90mkl==1.2.0'],
+    'mxnet': ['mxnet==1.2'],
```

Comment:
Whats the difference between this and mxnet_cpu?

Reply:
mxnet-mkl is intel mkl
vanilla mxnet uses openBLAS which works better on AMD cpus correct that is the difference correct?